draft-specifications/CapTP Specification.md

```diff
@@ -321,80 +322,82 @@ These three peers imply three sessions connecting them:
 
 When a message is sent over a CapTP boundary that includes a [Reference][Model-Reference]
 imported from a different session, a handoff MUST occur. The handoff is
-initated by the Gifter doing two things:
-
-- Depositing a "gift" to the exporter's bootstrap object.
-- Creating and sending a signed `desc:handoff-give` to the Receiver in place of the [Reference][Model-Reference].
+initated by the Gifter [depositing-the-gift](#depositing-the-gift) with the
+Exporter and [referencing the gift](#referencing-the-gift) with the Receiver,
+*in any order*.
 
 ### [Depositing the gift](#depositing-the-gift)
 
 The gift is simply the imported [Reference][Model-Reference] that the Gifter holds. This is
 deposited by sending a [`op:deliver-only`](#opdeliver-only) message to the
-exporter's bootstrap object with three `args`, which are:
+Exporter's bootstrap object with three `args`, which are:
 
-1.  A symbol that is the value `deposit-gift`
-2.  A non-negative integer that is the ID of the gift, this MUST not have been used
-    before in the **Gifter-Exporter** session (the `gift-id`)
-3.  The [Reference][Model-Reference] being exported to the Receiver.
+1.  A symbol whose value is "deposit-gift"
+2.  A previously-unused non-negative integer gift ID
+3.  The [Reference][Model-Reference] being exported.
 
 This is an example of how the message would look like:
 
 ```text
 <op:deliver-only <desc:export 0>               ; Remote bootstrap object
                  ['deposit-gift                ; Symbol "deposit-gift"
                   42                           ; gift-id, a non-negative integer (>=0)
-                  <desc:import-object ...>]>   ; remote object being shared
+                  <desc:import-object ...>]>   ; reference being shared
```

Comment:
should not lose qualification that the reference must be remote, must not be local.